wasm/src/lib.rs

```diff
@@ -32,7 +32,7 @@ fn run(config: &str, git_callback: js_sys::Function) -> Result<(), Box<dyn std::
 
     let mut changelog = Changelog::new(&mut template, git);
 
-    changelog.generate()?;
+    changelog.generate(None)?;
```

Comment:
Yeah, this is fine since the WASM module currently only supports `check` command.